app/organizers/reopen_authorization.rb

```diff
@@ -5,5 +5,5 @@ class ReopenAuthorization < ApplicationOrganizer
     context.authorization_request = context.authorization.request
   end
 
-  organize ExecuteAuthorizationRequestTransitionWithCallbacks
+  organize TransitionAuthorizationRequestToPreviousStage, ExecuteAuthorizationRequestTransitionWithCallbacks
```

Comment:
```suggestion
  organize TransitionAuthorizationRequestToPreviousStage, 
    ExecuteAuthorizationRequestTransitionWithCallbacks
```

c'est plus lisible et consistant avec le reste